packages/shared/src/index.ts

```diff
@@ -26,5 +26,6 @@ export * from "./lib/abstractBoard/intersection";
 export * from "./variants/s_fractional";
 export { count_binary_ones } from "./variants/thue_morse";
 export { sFractionalMoveColors } from "./variants/s_fractional";
+export * from "./variants/deprecated/badukWithAbstractBoard";
```

Comment:
I'm a little surprised this needs to be exported now that everything had been moved out of here. Is this used in vue-client?

Reply:
It's currently used in BadukBoardAbstract.vue <- this component only exists for mapping the config of legacy games of this variant to a Baduk config. So I'm thinking that we can add the mapping function as sanitizeConfig in the variant interface of BadukWithAbstractBoard, and then remove the above board component, as well as the export in index.ts.

Should I do it in this PR or a new one?

Reply:
Separate PR would be fine! Mainly just curious how it gets used